worldengine/drawing_functions.py

```diff
@@ -502,13 +502,13 @@ def _dynamic_draw_a_mountain(pixels, rng, x, y, w=3, h=3):
         darkarea = int(bottomness * w / 2)
         lightarea = int(bottomness * w / 2)
         for itx in range(darkarea, leftborder + 1):
-            pixels[x - itx, y + mody] = gradient(itx, darkarea, leftborder,
+            pixels[y + mody, x - itx] = gradient(itx, darkarea, leftborder,
                                                  (0, 0, 0), (64, 64, 64))
         for itx in range(-darkarea, lightarea + 1):
-            pixels[x + itx, y + mody] = gradient(itx, -darkarea, lightarea,
+            pixels[y + mody, x - itx] = gradient(itx, -darkarea, lightarea,
                                                  (64, 64, 64), (128, 128, 128))
         for itx in range(lightarea, leftborder):
-            pixels[x + itx, y + mody] = (181, 166, 127, 255)  # land_color
+            pixels[y + mody, x - itx] = (181, 166, 127, 255)  # land_color
```

Comment:
Should I maybe just remove the whole function? It is not used, not finished and (probably) soon will be obsolete due to https://github.com/Mindwerks/worldengine/issues/177.